spyder/widgets/calltip.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, parent=None, as_tooltip=False):
         """
         Shows tooltips that can be styled with the different themes.
         """
-        super(ToolTipWidget, self).__init__(parent, Qt.ToolTip)
+        super(ToolTipWidget, self).__init__(None, Qt.ToolTip)
```

Comment:
Why is this needed?

Reply:
Without this the tooltips position sometimes is incorrectly set. While testing, this was the way I found to make the position calculation to work correctly. I think that maybe the `Qt.ToolTip` window flag has something to do with this behavior too but I'm unsure.

Also, I choose to try this because the `CallTipWidget` does the same thing ([sets the parent to `None` when calling the super `__init__`](https://github.com/spyder-ide/spyder/blob/5b9d1e98e54c4a44f18f454d6f1169e7a57e49e3/spyder/widgets/calltip.py#L257))
```

Reply:
Ok, thanks @dalthviz for the explanation. If we're already doing that for `CallTipWidget`, then we're on the safe side doing this here too.